.travis.yml

```diff
@@ -70,10 +72,21 @@ matrix:
 
 before_install:
   - phpenv config-rm xdebug.ini
+  #- nvm install node 8.9.0
```

Comment:
This was the old way to use node as there was an issue with versions back in November. It is no longer the case